src/main/java/com/gildedrose/BackstagePass.java

```diff
@@ -1,33 +1,35 @@
 package com.gildedrose;
 
-public class BackstagePass {
+public class BackstagePass implements Ageable {
     private int qualityIncrementation;
+    private final Item item;
 
-    public BackstagePass() {
+    public BackstagePass(Item item) {
+        this.item = item;
         this.qualityIncrementation = 1;
     }
 
-    public void age(Item item) {
-        item.quality = updateQuality(item.quality, item.sellIn);
-        item.sellIn = updateSellIn(item.sellIn);
+    public void age() {
+        item.quality = updateQuality();
+        item.sellIn = updateSellIn();
     }
 
-    private int updateQuality(int quality, int sellIn) {
-        if (sellIn <= 10
-                && sellIn > 5) {
+    private int updateQuality() {
+        if (item.sellIn <= 10
+                && item.sellIn > 5) {
             this.qualityIncrementation = 2;
-        } else if (sellIn <= 5
-                && sellIn > 0) {
+        } else if (item.sellIn <= 5
+                && item.sellIn > 0) {
             this.qualityIncrementation = 3;
-        } else if (sellIn == 0) {
-            this.qualityIncrementation = -(quality);
+        } else if (item.sellIn == 0) {
+            this.qualityIncrementation = -(item.quality);
         }
-        quality += qualityIncrementation;
-        return Math.min(quality, 50);
+        item.quality += qualityIncrementation;
+        return Math.min(item.quality, 50);
```

Comment:
**Suggestion**: Possible to attach this number to a named variable for context, instead of having it be a magic number?

Reply:
Addressed in [this commit](https://github.com/jramnani/GildedRoseJavaLLC/pull/19/commits/9eb997feb4c13fe890e16515432c1014649f69ed)